library/Mockery/Adapter/Phpunit/MockeryPHPUnitIntegration.php

```diff
@@ -22,19 +22,37 @@
 
 use Mockery;
 
+if (\PHP_VERSION_ID < 70000) {
```

Comment:
This might break existing PHPUnit 6 and 7 tests, PHPUnit 8+ should be explicitly targeted instead.